Weird error messages

### What version of Turborepo are you using?

1.1.3-canary.2

### What package manager are you using / does the bug impact?

Yarn v2/v3 (node_modules linker only)

### What operating system are you using?

Mac

### Describe the Bug

When a run command fails, Turborepo outputs a weird error message.
```
@flux-ui/checkbox:build: src/checkbox.tsx(45,7): error TS2769: No overload matches this call.
@flux-ui/checkbox:build:   Overload 1 of 3, '(props: Omit<CheckboxProps & RefAttributes<HTMLButtonElement>, "checked" | "css" | "as"> & TransformProps<{ checked?: boolean | "false" | undefined; }, {}> & { ...; }): ReactElement<...> | null', gave the following error.
@flux-ui/checkbox:build:     Type 'CheckedState' is not assignable to type 'boolean | "false" | ({ "@initial"?: boolean | "false" | undefined; } & { [x: string]: boolean | "false" | undefined; }) | undefined'.
@flux-ui/checkbox:build:       Type '"indeterminate"' is not assignable to type 'boolean | "false" | ({ "@initial"?: boolean | "false" | undefined; } & { [x: string]: boolean | "false" | undefined; }) | undefined'.
@flux-ui/checkbox:build:   Overload 2 of 3, '(props: Omit<CheckboxProps & RefAttributes<HTMLButtonElement>, "checked" | "css" | "as"> & TransformProps<{ checked?: boolean | "false" | undefined; }, {}> & { ...; }): ReactElement<...> | null', gave the following error.
@flux-ui/checkbox:build:     Type 'CheckedState' is not assignable to type 'boolean | "false" | ({ "@initial"?: boolean | "false" | undefined; } & { [x: string]: boolean | "false" | undefined; }) | undefined'.
@flux-ui/checkbox:build:   Overload 3 of 3, '(props: Omit<CheckboxProps & RefAttributes<HTMLButtonElement>, "checked" | "css"> & TransformProps<{ checked?: boolean | "false" | undefined; }, {}> & { ...; }): ReactElement<...> | null', gave the following error.
@flux-ui/checkbox:build:     Type 'CheckedState' is not assignable to type 'boolean | "false" | ({ "@initial"?: boolean | "false" | undefined; } & { [x: string]: boolean | "false" | undefined; }) | undefined'.
@flux-ui/checkbox:build: 
@flux-ui/checkbox:build: [CHECKBOX] DTS Build error
@flux-ui/checkbox:build: Error: Failed to compile. Check the logs above.
@flux-ui/checkbox:build:     at error (/Users/johnhouse/Documents/flux-ui/node_modules/rollup/dist/shared/rollup.js:160:30)
@flux-ui/checkbox:build:     at throwPluginError (/Users/johnhouse/Documents/flux-ui/node_modules/rollup/dist/shared/rollup.js:21828:12)
@flux-ui/checkbox:build:     at Object.error (/Users/johnhouse/Documents/flux-ui/node_modules/rollup/dist/shared/rollup.js:22548:20)
@flux-ui/checkbox:build:     at Object.error (/Users/johnhouse/Documents/flux-ui/node_modules/rollup/dist/shared/rollup.js:22005:42)
@flux-ui/checkbox:build:     at Object.transform (/Users/johnhouse/Documents/flux-ui/node_modules/tsup/dist/rollup.js:8372:20)
@flux-ui/checkbox:build:     at /Users/johnhouse/Documents/flux-ui/node_modules/rollup/dist/shared/rollup.js:22793:37
@flux-ui/checkbox:build: Error: command finished with error: command (packages/checkbox) yarn yarn run build exited (1)
command (packages/checkbox) yarn yarn run build exited (1)

 Tasks:    1 successful, 2 total
Cached:    0 cached, 2 total
  Time:    4.649s 
```

### Expected Behavior

Expected error message to be `Error: command finished with error: command (packages/checkbox) yarn run build exited (1)` or something like that.

### To Reproduce

Force an error in a run command.

--- a/cli/internal/process/child.go
+++ b/cli/internal/process/child.go
@@ -15,7 +15,6 @@ import (
 	"math/rand"
 	"os"
 	"os/exec"
-	"path"
 	"strings"
 	"sync"
 	"syscall"
@@ -108,7 +107,9 @@ type NewInput struct {
 // sending signals to the child process, restarting the child process, and
 // gracefully terminating the child process.
 func newChild(i NewInput) (*Child, error) {
-	label := fmt.Sprintf("(%v) %v %v", i.Cmd.Dir, path.Base(i.Cmd.Path), strings.Join(i.Cmd.Args, " "))
+	// exec.Command prepends the command to be run to the arguments list, so
+	// we only need the arguments here, it will include the command itself.
+	label := fmt.Sprintf("(%v) %v", i.Cmd.Dir, strings.Join(i.Cmd.Args, " "))
 	child := &Child{
 		cmd:         i.Cmd,
 		timeout:     i.Timeout,

